data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+			label decline
+			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do than. At least thank you for the stories." She leaves.`
+				decline
+	on complete
+		set "coalition: struggling author - beyond the hyperdrive"
+		conversation
+			`Anna gives you the book, and then you leave the ship. The moment you are out, three normal Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. It is broken as their translation box speaks.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
+			`	The Coalition is already familiar with you, so you tell them your name while Anna introduces herself. "About what, do you write?" asks the Kimek. "Oh, I write science fiction books," replies Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but reminds you that she will not be allowed to do much unless she registers. "Register?" asks Anna.`
+			`	"You have to register yourself with the Coalition before you can do much here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`
+			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`
+
+
+mission "Coalition: Struggling Author 2"
+	name "Help Anna Register"
+	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	to offer
+		has "Coalition: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Anna. She is typing away on her laptop at a table in the middle of a public square.`
+			`	"Oh, hello again <first>!" Anna says. "Is the ship ready?"`
+			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
+			choice
+				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
+				`	(Say nothing.)`
+			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe.`
```

Comment:
Again, this is a bit short. Maybe another paragraph about how she keeps staring at it/starts typing or writing/does something else/goes back to sleep?